src/aioswitcher/device/__init__.py

```diff
@@ -321,6 +346,23 @@ class SwitcherShutterBase(ABC):
     direction: ShutterDirection
 
 
+@dataclass
+class SwitcherSingleShutterDualLightBase(ABC):
+    """Abstraction for all switcher devices controlling shutter with dual light.
+
+    Args:
+        position: the current position of the shutter (integer percentage).
+        direction: the current direction of the shutter.
+        light: the current light state.
+        light2: the current second light state.
+    """
+
+    position: int
+    direction: ShutterDirection
+    light: LightState
+    light2: LightState
```

Comment:
Maybe it would be better to make this an array?

Reply:
It is possible but I think it makes things more complicated and not that much readable if you consider the maximum number of lights possible in a device is 3 using the product: Switcher Light SL03

Reply:
So when you add the 3rd light it will be `light3`? wouldn't it be better to have an array, than when you add additional light to the array it will just expand by one?

I leave @TomerFi to suggest what he thinks would be better here

Reply:
I think an array is unneeded here and can make even more confusions, but let's leave it for @TomerFi

Reply:
What is the chance we'll need to add another light slot? If we add more, I agree with @thecode: Let's switch to an array. But if it's a long shot, we can stick to vars, maybe change the name to "lightSlot1" and "lightSlot2" or something like that because light+light2 feels like a patch.

Another option might be to have a dict with a string key ("light_slot1", "light_slot2") and a State instance as a value.

Reply:
We will need another slot in the future see https://github.com/TomerFi/aioswitcher/pull/767#discussion_r1615241270, Switcher Light SL03 has 3 lights.

Reply:
Gotcha.
Let's do an array. it feels right.

Reply:
I changed that to an array.